docker/diffusers-pytorch-compile-cuda/Dockerfile

```diff
@@ -42,7 +42,6 @@ RUN python3.9 -m pip install --no-cache-dir --upgrade pip && \
         tensorboard \
         transformers \
         omegaconf \
-        pytorch-lightning \
-        xformers
+        pytorch-lightning
```

Comment:
Actually why do we have pytorch-lightning in here? We never use this library IMO. Can we remove it from all places in the dockers?

Reply:
Sounds good. I'll make the change.